lace/lace_cc/src/component.rs

```diff
@@ -12,7 +12,7 @@ use crate::traits::{LaceDatum, LaceLikelihood, LaceStat};
 
 /// Maintains a component model and a sufficient statistic capturing the data
 /// assigned to the component.
-#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
```

Comment:
Curious why you removed `PartialEq` from this?

Reply:
I can fix that.